CPC-CHARTER.md

```diff
@@ -237,27 +237,32 @@ CPC Chairperson or Director may serve.
 ## Section 8. Board Representation
 
 There are 3 Board seats earmarked for representation from the
-OpenJS Foundation projects.  The first of these board seats is filled by
-the CPC Director as outlined in Section 7 on [Elections][] and is
-always in place.  
+OpenJS Foundation projects.
```

Comment:
I'm  not sure this is quite right, since one is to represent the CPC, the second projects and the third the supporters representative.

It might be better to be more general. Possibly "OpenJS Foundation stakeholders"